StepDefinitions/LanguageTestStepDefinitions.cs

```diff
@@ -0,0 +1,91 @@
+using MarsQA.Helpers;
+using MarsQA.Pages;
+
+namespace MarsSpecFlow.StepDefinitions
+{
+    [Binding]
+    public class LanguageTestStepDefinitions : Driver
+    {
+        LanguagePage languagePage = new LanguagePage();
```

Comment:
Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors

Reply:
Moved the languagePage  initialization to constructor